dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -118,14 +119,12 @@ const StandFirstGrid = ({ children }: { children: React.ReactNode }) => (
 						'standfirst'
 						'lines'
 						'meta';
+					grid-column-gap: 0px;
```

Comment:
What's happening with this change?

Reply:
I couldn't understand why this is happening, but right now in prod for below desktop 20px gap is causing this:

![image](https://user-images.githubusercontent.com/15894063/145401431-afb5c17a-3ddc-4762-952d-bacfcf2d58a0.png)

And setting it to 0 would change the number of grid areas

![image](https://user-images.githubusercontent.com/15894063/145401768-8be96ef8-3a76-45e4-92f6-cf14db4c54a4.png)

Reply:
I seee... Nice spot!